### PR TITLE
Add note about restrictions on rescoring problems

### PR DESCRIPTION
**Pull request by srpearce**
Ticket: [DOC-1833](https://openedx.atlassian.net/browse/DOC-1833)

@mhoeber or @lamagnifica Please review.

I didn't update the change logs here because I plan to work on several more tickets today. I'll create a separate PR to update the change logs for the ones that get merged.

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/running_course/course_grades.rst [hunk 1]
```diff
@@ -511,22 +511,25 @@ change is unavoidable, you can make the following adjustments.
 
 * Rescore the submitted answer to reevaluate student work on a problem. You
   can rescore a problem for a single learner or for all of the students
-  enrolled in the course. See :ref:`rescore`.
+  enrolled in the course. 
+
+  Note that you cannot rescore problems in some cases if your problem has
+  multiple answer fields. For more information, see :ref:`rescore`.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/course_grades.rst:517]: suggest putting this into the "Note:" format or rewording to eliminate the "note", like 
For problems that have multiple answer fields, the option to rescore is not available in all cases

File: en_us/shared/building_and_running_chapters/running_course/course_grades.rst [hunk 2]
```diff
@@ -580,10 +583,19 @@ change to these values, you can rescore any responses that were already
...
 single student, or rescore the submissions made by every enrolled student.
 
-.. note:: You can only rescore problems that have a correct answer entered in 
+.. note:: 
+ You can only rescore problems that have a correct answer entered in 
  edX Studio. This procedure cannot be used to rescore problems that are scored
  by an external grader.
 
+ Additionally, you cannot rescore a problem if the problem has multiple
+ response fields and you have performed any of the following actions.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/course_grades.rst]: performed --> completed (or other verb, "perform" is notoriously hard to translate well)

Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/course_grades.rst]: just out of curiosity, what happens? do you get an error message or does it just fail?
and, does this apply only if you make changes after the problem is live and has some answers, or at any time?
(this one is really cool!)

Comment (srpearce) [inline en_us/shared/building_and_running_chapters/running_course/course_grades.rst]: @adampalay Can you answer Alison's questions above? (I swear I commented on this PR a few days ago, and GitHub ate it. Aargh.)

Comment (adampalay) [inline en_us/shared/building_and_running_chapters/running_course/course_grades.rst]: I can't remember exactly... @Shrhawk , do you remember? I think we run into an error and the job fails.

**Pull request merged**